python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -726,18 +721,32 @@ def write_message(self, message: PipesMessage) -> None:
 DAGSTER_PIPES_MESSAGES_ENV_VAR = "DAGSTER_PIPES_MESSAGES"
 
 
-class PipesEnvVarParamsLoader(PipesParamsLoader):
-    """Params loader that extracts params from environment variables."""
+class PipesSourceParamsLoader(PipesParamsLoader):
```

Comment:
`PipesMappingSourceParamsLoader` I think is more clear

Reply:
Loading by definition implies some "source", so IMO it doesn't add info as a descriptor. What about just `PipesMappingParamsLoader`?